Dockerfile

```diff
@@ -1,12 +1,13 @@
 ARG GDAL=ubuntu-small-3.3.3
 FROM osgeo/gdal:${GDAL} AS gdal 
 ENV LANG="C.UTF-8" LC_ALL="C.UTF-8"
+ARG PYTHON_VERSION=3.8
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    python3 python3-pip python3-dev python3-venv cython3 g++ && \
```

Comment:
Intentionally didn't change `python3-pip` as it raised a `package not found` error.